jest.config.js

```diff
@@ -5,6 +5,9 @@ const { constants } = require('jest-config');
  */
 module.exports = {
   setupFilesAfterEnv: ['@xstate-repo/jest-utils/setup'],
+  transformIgnorePatterns: [
+    'node_modules/(?!(@open-wc|lit-html|lit-element|lit|@lit)/)'
+  ],
```

Comment:
why do we need this? does it turn off ESM->CJS transform?

Reply:
It is necessary to use Jest with Lit.
You can see more information here.

https://github.com/jestjs/jest/issues/11783#issuecomment-1107861603

===
<img width="813" alt="jest-and-lit" src="https://github.com/statelyai/xstate/assets/3649029/0a6b7723-2bd2-41fa-a06a-5384524bdb0a">

Reply:
Jest still doesn't support modules??